star_tides/api/decorators/login_required.py

```diff
@@ -1,5 +1,34 @@
 ''' star_tides.api.decorators.login_required
 '''
 
-
 # TODO Implement a decorator to enforce login
+from functools import wraps
+from star_tides.api.controllers.base_controller import Controller
+import time
+
+
+def login_required(func):
+    ''' This decorator enforces that a request is authenticated with the server
+        in any way other than unauthenticated.
+
+        Args:
+            func - function
+
+        Returns: Should return responses
+    '''
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        now = int(time.time())
```

Comment:
Very much up to you: this could be a severe Java-ism but do we want some clock factory to make this more easier to test?

Reply:
Javaism. Lmao